src/app/api/crud/accesses.py

```diff
@@ -72,7 +72,7 @@ async def create_accessed_entry(
     return entry
 
 
-async def update_accessed_entry(table: Table, accesses: Table, entry_id: int, payload: Any):
+async def update_accessed_entry(table: Table, accesses: Table, entry_id: int, payload: Any, only_provided=False):
```

Comment:
Maybe add a UTest?
IMHO the default boolean should be set to True :)

Reply:
I changed the argument to `only_specified` and set its default value to True